near-contract-standards/src/access/mod.rs

```diff
@@ -1,129 +1,279 @@
+//! Based on [`openzeppelin/access`](https://github.com/OpenZeppelin/openzeppelin-contracts/tree/master/contracts/access) files.
+
 use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
 use near_sdk::{env, require, AccountId};
 use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
 
+pub type RoleId = [u8; 32];
+
+/// Contract module which provides a basic access control mechanism, where
+/// there is an account (an [`Self::owner`]) that can be granted exclusive access to
+/// specific functions.
+///
+/// By default, the owner account will be the [one that deploys](env::predecessor_account_id())
+/// the contract. This can later be changed with [`Self::transfer_ownership()`].
+///
+/// This module makes available the modifier [`Self::only_owner()`], which can be applied to your
+///  functions to restrict their use to the owner.
 #[derive(BorshSerialize, BorshDeserialize)]
 pub struct Ownable {
-    pub owner: AccountId,
-    pub default_owner: AccountId,
+    pub owner: Option<AccountId>,
```

Comment:
Added an `Option` to represent the intention that it can get into a state where it's no longer owned.  

Cons: requires more space for the `Option` itself. 
If we had a `NonZeroAccountId`, it could be made more efficient I think.

But anyway this is a possibility, as the 64x0 AccountId (which should come from a hash) should be pratically impossible to get.

Reply:
Good idea, better to have an option.